soccer/planning/RRTPlanner.cpp

```diff
@@ -23,32 +25,34 @@ Geometry2d::Point Planning::randomPoint()
 	return Geometry2d::Point(x, y);
 }
 
-RRTPlanner::RRTPlanner()
-{
-	_maxIterations = 100;
-}
+RRTPlanner::RRTPlanner(): _maxIterations(100)
```

Comment:
Let's get rid of this constructor and just supply the default value of 100 to the other one.